stock_picking_consolidation_priority/models/stock_move.py

```diff
@@ -121,4 +121,17 @@ def _consolidate_priority(self):
                 [[("id", "in", move_ids)], self._consolidate_priority_domain()]
             )
         )
-        moves.picking_id.write(self._consolidate_priority_values())
+        for move in moves:
+            # Flag the move as urgent.
+            move.write(self._consolidate_priority_values())
+            # Flag the picking as urgent if it only contains urgent moves.
+            picking = move.picking_id
+            picking_moves = picking.move_lines
+            if (
+                all(
+                    m.priority == self._consolidate_priority_value
+                    for m in picking_moves
```

Comment:
forgot to exclude the cancel / done moves

Reply:
Isn't that already done in `_consolidate_priority_domain`?